bundles/org.openhab.automation.jsscripting/README.md

```diff
@@ -800,6 +800,13 @@ actions.notificationBuilder('Hello World!').logOnly().send();
 // Sends a simple log notification with icon and tag
 actions.notificationBuilder('Hello World!').logOnly()
   .withIcon('f7:bell_fill').withTag('important').send();
+
+//sends a notification about a temperature change...
+actions.notificationBuilder(str).withIcon("oh:temperature").withTag("Temperature change").withReferenceId("living room").send();
```

Comment:
Use single quotes `'` instead of double quotes `"`.
Please provide a message, because this example won't work with `str` as argument to the notification builder.
And I would prefer to have the reference id without any spaces.